### ISSUE TITLE
Powered by Algolia toggle

### ISSUE THREAD

**Issue by diedexx** [algolia/algoliasearch-wordpress#333]
I get it, you don't want us to remove the "powered by Algolia" bit, but this really confused me 😅 .

class-algolia-settings.php:176

``` php
public function is_powered_by_enabled() {
        return ! defined( 'ALGOLIA_POWERED_BY' ) || ALGOLIA_POWERED_BY === false;
        /*$enabled = get_option( 'algolia_powered_by_enabled', 'yes' );

        return $enabled === 'yes';*/
    }
```

Using `ALGOLIA_POWERED_BY === true` here would make more sense to me. Also, will the option stored in the database be used in the (near) future? 
### Technical info
- WordPress version: 4.6.1
- Algolia Search plugin version: 1.1.0

**Comment by rayrutjes** [comment]
Hi @diedexx ,

I get your feeling, and I feel the same ;)

For now I didn't find the right place to add this option, but will for sure make this option editable from the Admin UI.

Algolia asks users to stick with the powered by logo on hacker plans (free plans), that is why we didn't simply add it to the settings page for now.

I think the best place to add this for now is in the Settings section of WordPress.

I'll leave this open until it's fixed ;) Probably 1.3 release.

Thx a lot for reporting that one ❤️

**Comment by rayrutjes** [comment]
We have for now very few issues related to removing the powered by branding.
Also, the powered by is mandatory for free plans and we are afraid that making it too easy to remove would be misleading for our users.

For now we will redirect paying users to the documentation about the constant that allows to switch powered by off.
